internal/artifacts/module_arch.go

```diff
@@ -187,6 +187,10 @@ func convert(data []interface{}) []string {
 }
 
 func isArchive(path string) bool {
+	entry, err := os.Stat(path)
```

Comment:
Change packModule function accordingly (to only check isArchive)

Reply:
done (with error)